code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -7,10 +9,58 @@
 	throw_range = 5
 	w_class = W_CLASS_TINY
 	flags = FPRINT
-	var/uses = 5
-	var/temp = null
-	var/max_uses = 5
+
+	var/list/available_spells = list(
+	/spell/targeted/projectile/magic_missile,
+	/spell/targeted/projectile/dumbfire/fireball,
+	/spell/lightning,
+	/spell/aoe_turf/disable_tech,
+	/spell/aoe_turf/smoke,
+	/spell/targeted/genetic/blind,
+	/spell/targeted/subjugation,
+	/spell/targeted/mind_transfer,
+	/spell/aoe_turf/conjure/forcewall,
+	/spell/aoe_turf/blink,
+	/spell/area_teleport,
+	/spell/targeted/genetic/mutate,
+	/spell/targeted/ethereal_jaunt,
+	/spell/aoe_turf/fall,
+	/spell/aoe_turf/knock,
+	/spell/targeted/equip_item/horsemask,
+	/spell/targeted/equip_item/clowncurse,
+	/spell/targeted/equip_item/frenchcurse,
+	/spell/targeted/shoesnatch,
+	/spell/targeted/equip_item/robesummon,
+	/spell/targeted/flesh_to_stone,
+	/spell/targeted/buttbots_revenge,
+	/spell/aoe_turf/conjure/pontiac,
+	/spell/noclothes
+	)
+
+	//Unlike the list above, the available_artifacts list builds itself from all subtypes of /datum/spellbook_artifact
+	var/list/available_artifacts = list()
+
+	var/uses = STARTING_USES
+	var/max_uses = STARTING_USES
+
 	var/op = 1
+	var/temp = null
```

Comment:
Since it looks like you're not using this for anything anymore, you should remove it.